plugin/src/lib.rs

```diff
@@ -187,6 +187,8 @@ impl<'a, 'cx> MutatorPlugin<'a, 'cx> {
 
                     mut_expression = quote_expr!(self.cx,
                                     {
+                                        ::mutagen::report_coverage();
```

Comment:
Why don't we add `$n` as an argument? Or even `$n, $n + x` (for some value of x if we have multiple mutations)? Because this way, we can not only detect whether mutations have been active at all, but also which mutations are covered, allowing us to restrict the set of tests to run per mutation even further.

Reply:
Added on: f9e543c8b39396d3a75548097804cbc6e6523c06